x-pack/plugins/task_manager/server/task_pool.ts

```diff
@@ -168,7 +168,7 @@ export class TaskPool {
         // we asssume the underlying task has been deleted while it was running
         // so we will log this as a debug, rather than a warn
         const errorLogLine = `Task ${taskRunner.toString()} failed in attempt to run: ${
-          err.message
+          err.message || err.error.message
```

Comment:
209 conflict error returns a nested error object, `err.message` returns undefined. i had to add this line to see the real error.